fundingmanager.go

```diff
@@ -322,6 +333,12 @@ var (
 	// of being opened.
 	channelOpeningStateBucket = []byte("channelOpeningState")
 
+	// openChanAnnPrefBucket is the database bucket used to store each
+	// channel's announcement preference signalled by the LSB of
+	// channel_flags of the open_channel message in the funding workflow.
+	// It only stores open channels' announcement preferences.
+	openChanAnnPrefBucket = []byte("open_chan_ann")
```

Comment:
Minor nit: the bucket name should be modified to match the styling of the bucket above (so change to camel case).